integrationTests/testProcessorNodeWithMultisigner.go

```diff
@@ -230,13 +230,13 @@ func CreateNodesWithNodesCoordinatorFactory(
 	}
 
 	epochsConfig := config.EnableEpochs{
-		StakingV2EnableEpoch:                     UnreachableEpoch,
-		ScheduledMiniBlocksEnableEpoch:           UnreachableEpoch,
-		MiniBlockPartialExecutionEnableEpoch:     UnreachableEpoch,
-		RefactorPeersMiniBlocksEnableEpoch:       UnreachableEpoch,
-		StakingV4EnableEpoch:                     UnreachableEpoch,
-		StakingV4InitEnableEpoch:                 UnreachableEpoch,
-		StakingV4DistributeAuctionToWaitingEpoch: UnreachableEpoch,
+		StakingV2EnableEpoch:                 UnreachableEpoch,
+		ScheduledMiniBlocksEnableEpoch:       UnreachableEpoch,
+		MiniBlockPartialExecutionEnableEpoch: UnreachableEpoch,
+		RefactorPeersMiniBlocksEnableEpoch:   UnreachableEpoch,
+		StakingV4Step2EnableEpoch:            UnreachableEpoch,
+		StakingV4Step1EnableEpoch:            UnreachableEpoch,
```

Comment:
for code consistency: switch order L237 with L238?

Reply:
Right, changed as suggested